grouping/compile.m

```diff
@@ -51,7 +51,7 @@ function compile(gccVer)
 %mkoctfile --mex ./matlab/segmentation/mex_contour.cc  -I./include
 mkoctfile --mex ./matlab/segmentation/mex_contour_sides.cc  -I./include
 %mkoctfile --mex ./matlab/segmentation/mex_line_inds.cc  -I./include
-mkoctfile --mex ./matlab/segmentation/mex_nonmax_oriented.cc  -I./include
+mkoctfile --mex -L./ -lopencv_gpb ./matlab/segmentation/mex_nonmax_oriented.cc  -I./include

```

Comment:
I think this should be -L../../lib -lopencv_gpb  [to match line 57]

The original patch did not work, but that tweak did.  Many thanks for the help.